MFVNeutralino/python/NtupleCommon.py

```diff
@@ -256,9 +275,23 @@ def miniaod_ntuple_process(settings):
     #process.selectedPatMuons.cut = '' # process.jtupleParams.muonCut
     #process.selectedPatElectrons.cut = '' # process.jtupleParams.electronCut
 
+    # change made to use corrected MET
+    process.mfvTriggerFloats.met_src = cms.InputTag('slimmedMETs', '', 'Ntuple')
+    if not settings.is_mc:
+      process.mfvTriggerFloats.met_filters_src = cms.InputTag('TriggerResults', '', 'RECO')
+    process.mfvTriggerFloats.isMC = settings.is_mc
+    process.mfvTriggerFloats.year = settings.year
+
     process.mfvGenParticles.gen_particles_src = 'prunedGenParticles'
     process.mfvGenParticles.last_flag_check = False
 
+    #for splitSUSY
+    process.mfvGenParticles.lsp_id = 1000021
```

Comment:
I guess this should go into one of our configs related to the samples, if possible.

Reply:
Maybe we need to think about how to do this efficiently. We can discuss about that later. :)